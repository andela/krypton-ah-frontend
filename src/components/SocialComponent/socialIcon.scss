@import "../colors";
@import "../fonts";

.ui {

  &.image {
    cursor: pointer !important;
    width: 2.8rem !important;
  }

  &.images {
    margin: 0 auto !important;
    margin-top: 30px !important;
    text-align: center;
  }

}
