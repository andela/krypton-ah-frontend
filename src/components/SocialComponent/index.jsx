import React from 'react';
import { Image } from 'semantic-ui-react';
import facebook from './images/Facebook.svg';
import google from './images/Google+.svg';
import linkedin from './images/Linkedin.svg';
import twitter from './images/Twitter.svg';
import './socialIcon.scss';

function SocialIcons() {
  return (
    <div>
      <div className="center aligned column">
        <Image.Group size="tiny">
          <Image src={facebook} />
          <Image src={google} />
          <Image src={twitter} />
          <Image src={linkedin} />
        </Image.Group>
      </div>
    </div>
  );
}

export default SocialIcons;
