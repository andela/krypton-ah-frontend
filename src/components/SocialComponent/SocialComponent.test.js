import React from 'react';
import { shallow } from 'enzyme';
import SocialComponent from '.';

describe('Modal Social', () => {
  it('it should matches the snapshot', () => {
    const wrapper = shallow(<SocialComponent />);
    expect(wrapper).toMatchSnapshot();
  });
});
