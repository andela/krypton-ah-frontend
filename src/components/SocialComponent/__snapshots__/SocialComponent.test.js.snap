// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modal Social it should matches the snapshot 1`] = `
<div
  className="socialIconsHolder"
>
  <a
    href="https://krypton-ah-stage.herokuapp.com/api/v1/auth/facebook"
    key="1"
  >
    <div
      className="groupIcon"
    >
      <Icon
        as="i"
        className="socialIcon"
        name="facebook official"
        size="big"
      />
      <span>
        Facebook
      </span>
    </div>
  </a>
  <a
    href="https://krypton-ah-stage.herokuapp.com/api/v1/auth/google"
    key="2"
  >
    <div
      className="groupIcon"
    >
      <Icon
        as="i"
        className="socialIcon"
        name="google plus square"
        size="big"
      />
      <span>
        Google
      </span>
    </div>
  </a>
  <a
    href="https://krypton-ah-stage.herokuapp.com/api/v1/auth/linkedin"
    key="3"
  >
    <div
      className="groupIcon"
    >
      <Icon
        as="i"
        className="socialIcon"
        name="linkedin"
        size="big"
      />
      <span>
        Linkedin
      </span>
    </div>
  </a>
  <a
    href="https://krypton-ah-stage.herokuapp.com/api/v1/auth/"
    key="4"
  >
    <div
      className="groupIcon"
    >
      <Icon
        as="i"
        className="socialIcon"
        name="twitter square"
        size="big"
      />
      <span>
        Twitter
      </span>
    </div>
  </a>
</div>
`;
